lib/rom/sql/commands_ext/postgresql.rb

```diff
@@ -0,0 +1,19 @@
+module ROM
+  module SQL
+    module Commands
+      module PostgreSQL
```

Comment:
Would be better to use "Postgres" because that's how it's called in Sequel, that's the database adapter type in config files and it doesn't have "SQL" acronym inside which can cause trouble due to naming conventions etc.